ens-de-lyon-centre-d-ingenierie-documentaire.csl

```diff
@@ -109,7 +109,7 @@
                 <name-part name="family" font-variant="small-caps"/>
               </name>
               <et-al font-style="italic"/>
-              <label form="short" prefix="&#160;(" suffix=".)"/>
+              <label form="short" prefix=" (" suffix=".)"/>
```

Comment:
(if you aren't, could you put the escape codes back? We very easily lose unescaped special spaces, which often turn into regular spaces when we copy and paste style code around)